varcode/variant_collection.py

```diff
@@ -125,29 +125,33 @@ def variant_effects(self, raise_on_error=True):
         Parameters
         ----------
 
-        raise_on_error : bool, optional.
+        raise_on_error : bool, optional
             Raise exception if error is encountered while annotating
             transcripts, otherwise track errors in VariantEffect.errors
             dictionary (default=True).
+
         """
         return [
-            self.annot.describe_variant(variant, raise_on_error=raise_on_error)
+            self.annot.effect(
+                variant=variant,
+                raise_on_error=raise_on_error)
             for variant
             in self.records
         ]
 
-    def effects_to_string(self):
+    def effects_to_string(self, *args, **kwargs):
         """
         Create a long string with all transcript effects for each mutation,
         grouped by gene (if a mutation affects multiple genes).
         """
```

Comment:
a note in the docstring about where the args and kwargs are sent would be helpful